clusterloader2/testing/density/100_nodes/constraints.yaml

```diff
@@ -10,7 +10,7 @@ coredns:
   memoryConstraint: 31457280 #30 * (1024 * 1024)
 fluentd-gcp:
   cpuConstraint: 1
-  memoryConstraint: 241172480 #230 * (1024 * 1024)
+  memoryConstraint: 314572800 #300 * (1024 * 1024)
```

Comment:
I can live with increasing it, but I would like to understand what caused it.
This seems like a regression.

Did we start logging something more excessively? Did we changed fluentd version?

Reply:
Agreed, opened an issue to investigate - https://github.com/kubernetes/kubernetes/issues/77492